src/Raven.Server/Documents/Sharding/Queries/ShardedQueryProcessor.cs

```diff
@@ -80,7 +79,7 @@ public ShardedQueryProcessor(TransactionOperationContext context, ShardedQueries
 
     public long ResultsEtag => _result.ResultEtag;
 
-    public void Initialize()
+    public void Initialize(out Dictionary<int, BlittableJsonReaderObject> queryTemplates)
```

Comment:
I see why you had to do it via `out`. I will try to refactor this when doing https://issues.hibernatingrhinos.com/issue/RavenDB-19084/Sharding-Queries-ShardedQueryProcessor